data_cleaned/country_informations.csv

```diff
@@ -1,49 +1,3523 @@
-Country_source_CB
-Algeria
-Argentina
-Australia
-Azerbaijan
-Bahrain
-Bangladesh
-Botswana
-Brazil
-Bulgaria
-Canada
-China
-Colombia
-Denmark
-Germany
-Greece
-Guyana
-India
-Indonesia
-Iran
-Iraq
-Israel
-Kazakhstan
-Kuwait
-Libya
-Mexico
-Mongolia
-Mozambique
-Nigeria
-North Korea
-Norway
-Pakistan
-Poland
-Qatar
-Russia
-Saudi Arabia
-Serbia
-South Africa
-Syria
-Tanzania
-Turkmenistan
-Türkiye
-Ukraine
-United Arab Emirates
-United Kingdom
-United States
-Uzbekistan
-Venezuela
-Zimbabwe
+Country_source_CB,Year,Category,Series,Value,Footnotes,Source
+Algeria,2010,"Population, density and surface area",Population mid-year estimates (millions),35.86,,"United Nations Population Division, New York, World Population Prospects: The 2022 Revision, last accessed July 2022."

```

Comment:
I'm not sure but it seems, there is a formating error of the informations in the output csv. We can talk about it on Slack if you want

Reply:
Postporcess OK